src/app/garfile/cancel/post.controller.js

```diff
@@ -16,11 +16,15 @@ module.exports = (req, res) => {
       }).then(() => {
         req.session.successMsg = 'The GAR has been successfully cancelled';
         req.session.successHeader = 'Cancellation Confirmation';
-        res.redirect('/home');
+        req.session.save(() => {
```

Comment:
Not sure if this is necessary, but this definitely ensures the successMsg and successHeader get stored in the session before the redirect.